Assets/Scripts/SS3D/Networking/Events/NetworkConnectionStarted.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Coimbra.Services.Events;
+
+namespace SS3D.Networking.Events
```

Comment:
might remove this one as well

Reply:
Not sure why. This is a way to guarantee things will run only after the server started, if we don't want to use the NetworkBehaviour class.